storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -22,6 +22,10 @@ otherwise this won’t be read by a screen reader.
 Add an `aria-describedby` attribute to the Field Set
 and provide the `id` of the describing element as its value.
 
+Do not use rich text (such as links or lists) in descriptions.
+Some screen readers, like VoiceOver, skip over rich text when reading out descriptions.
+For more information, see [NL Design System](https://nldesignsystem.nl/richtlijnen/formulieren/descriptions/inhoud/).
+
```

Comment:
I like pasta, but this is a _lot_ of copy-pasta. I understand why you did it, but shouldn’t we find a slightly better way?

Can we add an `id` to the guideline on the `ErrorMessage` page and shorten all occurrences to something like this?

`Do not use rich text (such as links or lists) in an Error Message. [Read why](/ErrorMessage#no-rich-text)`

And the same for the description, probably with the source explanation on the page for `Field`?